packages/connection-form/src/components/connection-form-actions.tsx

```diff
@@ -184,7 +184,7 @@ export function ConnectionFormModalActions({
           variant={ButtonVariant.Primary}
           onClick={onSaveAndConnect}
         >
-          Save &amp; Connect
+          Connect
```

Comment:
Not touching anything else in this file, seems like we already have COMPASS-8098 to clean the implementation here